File: java/Mahasiswa.java
//Deklarasi kelas Mahasiswa
class Mahasiswa extends Human{
    //Private Attributes
    String NIM;
    String prodi;
    String fakultas;
    
    //Konstruktor
    public Mahasiswa(){
        //konstruktor kosong
    }

    public Mahasiswa(String nama, String NIM, String gender, String prodi, String fakultas){
        //konstruktor dengan parameter
        this.nama = nama;
        this.NIM = NIM;
        this.gender = gender;
        this.prodi = prodi;
        this.fakultas = fakultas;
    }

    //setter getter

    //SETTER
    public void setNIM(String NIM){
        this.NIM = NIM;
    }

    public void setprodi(String prodi){
        this.prodi = prodi;
    }

    public void setfakultas(String fakultas){
        this.fakultas = fakultas;
    }

    //GETTER

     public String getNIM(){
        return this.NIM;
    }

    public String getprodi(){
        return this.prodi;
    }

    public String getfakultas(){
        return this.fakultas;
    }

}

File: python/mahasiswa.py
from human import Human

class mahasiswa(Human):

    def __init__(self, NIK="", nama="", gender="", NIM = "", prodi = "", fakultas = ""):
        super().__init__(NIK, nama, gender)
        self.__NIM = NIM
        self.__prodi = prodi
        self.__fakultas = fakultas

    def getNIM(self):
        return self.__NIM

    def setNIM(self, NIM):
        self.__NIM = NIM

    def getprodi(self):
        return self.__prodi

    def setprodi(self, prodi):
        self.__prodi = prodi

    def getfakultas(self):
        return self.__fakultas

    def setfakultas(self, fakultas):
        self.__fakultas = fakultas

    